javascript/javascript3/week3/user.js

```diff
@@ -0,0 +1,233 @@
+class Users {
```

Comment:
You can think about the responsability of the classes you made. They might be bit to bit. It is hard to say what this class represents, it also has a very vague name (if I see something named Users, I'd expect some array of users :). I think you could split this one into two - three classes: one for `User`, `UsersCollection`, something to deal with forms maybe, maybe for handling requests... There is no one good way of splitting it, but generally a class should have one responsability - this is one of the pillars of object oriented programming. You can read more about it here https://duncan-mcardle.medium.com/solid-principle-1-single-responsibility-javascript-5d9ce2c6f4a5 This might be bit advanced, so do not worry if you don't understand at first try, it gets more clear with practice and some more work with data and how we can structure and reflect it in our code :)